generated/niswitch/session.py

```diff
@@ -2447,7 +2447,7 @@ def wait_for_scan_complete(self, maximum_time_ms=datetime.timedelta(milliseconds
         error.
 
         Args:
-            maximum_time_ms (int): Specifies the maximum length of time to wait for the switch module to
+            maximum_time_ms (float in seconds or datetime.timedelta): Specifies the maximum length of time to wait for the switch module to
```

Comment:
If this is a source breaker and not just a documentation fix please call out in CHANGELOG.

Reply:
This was a metadata bug being addressed. nidmm & niswitch were still using `'python_api_converter_type'` instead of `'python_type'`. Now all drivers are using `'type_in_documentation'`